contracts/DataFeedConsumer.sol

```diff
@@ -1,32 +1,30 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.16;
```

Comment:
@nick-bisonai We could also increase the solidity version to 0.8.20 since that is used in the v0.2.